content/docs/start/experiments/experiment-management.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Get Started: Experiment Management'
+description:
+  'Get started with managing experiments with DVC. Manage your experiments and
+  share them with others using the workflow that best suits your needs.'
+---
+
+# Get Started: Experiment Management
+
+After having compared all the experiments, ML Practitioners still need to agree
+on which one is the best and manage the remaining candidates.
+
+<abbr>DVC Experiments</abbr> are fully compatible with Git workflows, so you can
+manage the experiments using the workflow that best suits your needs.
+
+## Sharing
+
+Unless you have enabled
+[Studio Live Experiments](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots),
+the DVC experiments only exist in your repo and people can't manage or view them
+from other machines.
+
+To share an experiment with others, you can use `dvc exp push`, providing the
+names of the git remote and the experiment:
+
+TODO: VSCode sharing?
```

Comment:
Let's make it a follow-up since I don't think there is a VS Code option to do `dvc exp push` today.

@mattseddon For the future state, we will probably need to:
1. Add an option that does this (`dvc exp push` should automatically take care of sharing to studio).
2. Reconsider how to integrate the VS Code Studio settings with DVC Studio token handling.
3. Consolidate/drop existing sharing options if they aren't needed.

Reply:
On the other hand, we could show `Share to Studio` as an alternative for now, and it would make sense to show VS Code for the below sections on persisting and cleaning since they will be easier to do from VS Code than learning the CLI commands.

Reply:
> On the other hand, we could show Share to Studio as an alternative for now, and it would make sense to show VS Code for the below sections on persisting and cleaning since they will be easier to do from VS Code than learning the CLI commands.

Will do that and use tabs as in the first page